RELEASE-NOTES.md

```diff
@@ -1,3 +1,87 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
```

Comment:
What do you mean by disable here? Do you mean that the object cannot be updated?